sidebars.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
     {
       type: "category",
       label: "Release notes",
+      link: {type:"doc", id:"getting-started/release-notes"},
       items: [
         "getting-started/release-notes/release-notes-overview",
```

Comment:
And then try removing this from the list of items so it doesn't show up twice under the "Release notes" category on the sidebar:

```suggestion
```